bin/dev

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if ! command -v foreman &> /dev/null
+then
+  echo "Installing foreman..."
+  gem install foreman
+fi
```

Comment:
lol this didn't actually work for me and i had to do it manually

Reply:
dunno why!

Reply:
weird!!